src/libfuncs/bounded_int.rs

```diff
@@ -114,8 +115,13 @@ fn build_add<'ctx, 'this>(
     let compute_ty = IntegerType::new(context, compute_range.offset_bit_width()).into();
 
     // Zero-extend operands into the computation range.
-    assert!(compute_range.offset_bit_width() >= lhs_width);
-    assert!(compute_range.offset_bit_width() >= rhs_width);
+    if !compute_range.offset_bit_width() >= lhs_width {
+        native_panic!("the lhs_range bit_width must be less or equal than the compute_range");
+    }
```

Comment:
Please make a `native_assert` macro instead of this (in all instances).

Reply:
I added an implementation of native_assert on [this PR.](https://github.com/lambdaclass/cairo_native/pull/907) You could take it from there.

Reply:
Done!